.github/workflows/windows-qt6.yml

```diff
@@ -50,6 +50,10 @@ jobs:
         uses: hendrikmuhs/ccache-action@v1.2
         with:
           max-size: 1G
+          key: build-ccache-win64-qt6-${{ github.event.pull_request.base.ref || github.ref_name }}
+          save: ${{ github.event_name == 'push' }}
+          restore-keys: |
+            build-ccache-win64-qt6-master
```

Comment:
```suggestion
            build-ccache-win64-qt6-${{ github.event.pull_request.base.ref || github.ref_name }}
```

Reply:
This is copied from other workflows (mingw64), does it also need to be adjusted there?

Reply:
yes, otherwise you are pushing release_3-34 for instance but always pull master.